doc/nvnmd/nvnmd.md

```diff
@@ -255,24 +255,24 @@ where items are defined as:
 | job_name     | the name of computing job, which can be named freely                                                                       | a string       |
 | command      | the command to be executed on the computing node                                                                           | a string       |
 | log_file     | the log file that can be viewed at any time during the calculation process, which can be viewed on the Bohrium "Jobs" page | a string       |
-| machine_type | the machine type used for the job                                                                                          | "c8_m32_cpu"   |
+| machine_type | the machine type used for the job                                                                                          | "c1_m4_cpu", "c4_m16_cpu", "c8_m32_cpu"   |
 | job_type     | the job type                                                                                                               | "container"    |
 | image_name   | the image name used for the job                                                                                            | "lammps_dp:29Sep2021"|
-| platform     | resource provider                                                                                                          | "hnu"          |
+| platform     | resource provider                                                                                                          | "hnugba"          |
 | project_id   | the project ID to which the job belongs, which can be viewed on the "Projects" page                                        | a integer      |
 
-Notice：The task will use 8 CPU cores for computation, so do not repeatedly use the `mpirun` command, otherwise an error will be reported. All 0000 after "project_id" need to be replaced with your own project ID, which can be viewed on the "Projects" page. Also, the JSON file format requires that no commas be added after the last field within the {}, otherwise, there will be a syntax error.
+Notice：The task will use 4 CPU cores for computation, so do not repeatedly use the `mpirun` command, otherwise an error will be reported. All 0000 after "project_id" need to be replaced with your own project ID, which can be viewed on the "Projects" page. Also, the JSON file format requires that no commas be added after the last field within the {}, otherwise, there will be a syntax error. Please check the (documentation)[https://github.com/LiuGroupHNU/md-data/blob/master/code/doc/mdpu/hardware.md] for the latest hardware configuration information.
```

Comment:
Correct the Markdown grammar...
```suggestion
Notice：The task will use 4 CPU cores for computation, so do not repeatedly use the `mpirun` command, otherwise an error will be reported. All 0000 after "project_id" need to be replaced with your own project ID, which can be viewed on the "Projects" page. Also, the JSON file format requires that no commas be added after the last field within the {}, otherwise, there will be a syntax error. Please check the [documentation](https://github.com/LiuGroupHNU/md-data/blob/master/code/doc/mdpu/hardware.md) for the latest hardware configuration information.
```